Enhance signer setup guide for SPO (#1263)
input-output-hk/mithril: docs/website/root/manual/getting-started/run-signer-node.md

tail /var/log/syslog
```

### Rotating the KES keys

:::danger

> When the KES keys have expired, the Mithril signer is not able to register to the Mithril protocol.

Reviewer: ```suggestion
When the KES keys expire, the Mithril signer is unable to register with the Mithril protocol.
```

:::

> When you have rotated the KES keys on your Cardano block producer, we recommend the following upgrade procedure for your Mithril signer node:

Reviewer: ```suggestion
After rotating the KES keys on your Cardano block producer, we recommend following this upgrade procedure for your Mithril signer node:
```

1. Update the `KES_SECRET_KEY_PATH` entry of your environment file to reflect the location of the **new KES secret key file**.

> 2. Update the `OPERATIONAL_CERTIFICATE_PATH` entry of your environment file to reflect the location of the **new Operational certificate file**.

Reviewer: ```suggestion
2. Update the `OPERATIONAL_CERTIFICATE_PATH` entry of your environment file to reflect the location of the **new operational certificate file**.
```
